recipes/spice-utilities/meta.yaml

```
@@ -0,0 +1,368 @@
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/oem2spk  # [osx]
+    folder: utilities  # [osx]
+    sha256: fa5a5dfd2bee88f51ad5eb52af9d36f700c26d77a607b66a34ba4845b84c5c40  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/optiks  # [osx]
+    folder: utilities  # [osx]
+    sha256: 83aa6505079f28c7553a96bac88c12a6b4f8564a3da70eebd82851ecfa3aef8f  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/orbnum  # [osx]
+    folder: utilities  # [osx]
+    sha256: 293b88dfad418c0b908060796039a2e2e753dda1ef8c7e70326a1d4291bbb03f  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/pinpoint  # [osx]
+    folder: utilities  # [osx]
+    sha256: 802fdca8009fb394e45d32873339d71da218f5cdb2dc7411ca928c5651400007  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/prediCkt  # [osx]
+    folder: utilities  # [osx]
+    sha256: 53a6cd3b22a08666104087ed8bf11d3a249aefbd75e2771030855075c6e259fa  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spacit  # [osx]
+    folder: utilities  # [osx]
+    sha256: ec36d06bf00bc088408652ea0b61dcb44734602d46c99a4fde894ecc06f5fe88  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spk2oem  # [osx]
+    folder: utilities  # [osx]
+    sha256: f4a0af4633941159e11ef5961b78a1cf49883d089a85298e5375cfc2eb684d1e  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spkdiff  # [osx]
+    folder: utilities  # [osx]
+    sha256: 4cf959e55117deadbcc7e36cd8a250a36506d129f8185931cc5ebdaf08389fb7  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spkmerge  # [osx]
+    folder: utilities  # [osx]
+    sha256: fd903d40797d1f8d36e5bd78c4a403758421d328e8e8e9b110a660683604dad5  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/spy  # [osx]
+    folder: utilities  # [osx]
+    sha256: a6db357f4b8c85610f0835e8536650938943ce00c6b010462ecb8cfeb688fc30  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/starsb  # [osx]
+    folder: utilities  # [osx]
+    sha256: 27f22ab5b0da1b74cc5b6f8b955b8b2601b009e8c192c12760c9d12e3f50ce7a  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/tobin  # [osx]
+    folder: utilities  # [osx]
+    sha256: ecd64ceecacff5796f5108a63e265cbfe75345d7922d624cce8ced0b9786032a  # [osx]
+  - url: https://naif.jpl.nasa.gov/pub/naif/utilities/MacIntel_OSX_64bit/toxfr  # [osx]
+    folder: utilities  # [osx]
+    sha256: a5c7f8ed96eaaaa87c3b53f833bdf3b85100b89825fdf87415673696aaf9a5ac  # [osx]
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - /lib64/libc.so.6  # [linux64]
+    - /lib64/libm.so.6  # [linux64]
+
+test:
+  commands:
+    - chronos -h
+
+about:
+  home: https://naif.jpl.nasa.gov/naif/utilities.html
+  license: MIT
```

Not sure where the MIT comes from. The attached license calls itself "SPICE Software License", so I'm think this should be:

```suggestion
  license: SPICE Software License
```

---

I switched it back to MIT (see my other comment in the chat) because that's what the CSPICE feedstock has. But I don't know. I assume the CSPICE one was OK? (both that and this have the same license).

---

Also note that there's no code here. Like the CSPICE one, we are just downloading binaries from the JPL server.

---

Except that the license provided here is clearly not MIT because it puts limits on redistribution.